src/main/java/com/jfrog/ide/idea/ui/webview/WebviewResourceHandler.java

```diff
@@ -0,0 +1,109 @@
+package com.jfrog.ide.idea.ui.webview;
+
+import com.jfrog.ide.idea.log.Logger;
+import org.cef.callback.CefCallback;
+import org.cef.handler.CefLoadHandler;
+import org.cef.handler.CefResourceHandler;
+import org.cef.misc.IntRef;
+import org.cef.misc.StringRef;
+import org.cef.network.CefRequest;
+import org.cef.network.CefResponse;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.net.URLConnection;
+
+/**
+ * We use this handler to read web resources that are saved inside the plugin's JAR.
+ */
+public class WebviewResourceHandler implements CefResourceHandler {
+    private URL currUrl;
+    private URLConnection connection;
+    private InputStream inputStream;
+    private boolean error;
+
+    @Override
+    public boolean processRequest(CefRequest request, CefCallback callback) {
+        String pathToResource = request.getURL().replace("http://jfrog-idea-plugin", "/jfrog-ide-webview");
+        currUrl = WebviewResourceHandler.class.getResource(pathToResource);
+        try {
+            //noinspection DataFlowIssue
+            connection = currUrl.openConnection();
+            if (inputStream != null) {
+                inputStream.close();
+            }
+            inputStream = connection.getInputStream();
+            error = false;
+            callback.Continue();
+            return true;
+        } catch (IOException | NullPointerException e) {
+            Logger.getInstance().error("An error occurred while reading webview resource.", e);
```

Comment:
Should we also include the resource path (or name) for debugging purposes?

Reply:
Yes. Good idea, I'll add it.